migtests/tests/analyze-schema/dummy-export-dir/schema/tables/table.sql

```diff
@@ -16,27 +16,23 @@ CREATE TABLE sales (
 	PRIMARY KEY (bill_date)
 ) PARTITION BY RANGE (extract(year from date(bill_date))) ;
 
-
-CREATE TABLE t1 (
+-- this table has PK but not include all partition keys as PK, hence reported in analyze-schema
```

Comment:
keep the comments concise, and grammatically correct as a phrase, re-phrase this to:
`all partition keys are not included in the PK constraint; to be reported during analyze-schema`